frontend/src/features/help/forms/QuestionForm.tsx

```diff
@@ -34,8 +34,10 @@ const QuestionForm: React.FunctionComponent<QuestionFormProps> = ({ formValues,
       onSubmit={noop}
       validateOnMount={true}
       validate={(values: any) => {
-        const body = values.question;
-        const mailto = `mailto:${pimsSupportEmail}?subject=Question - ${formValues.page}&body=${body}`;
+        const mailto = {
```

Comment:
unfortunately coverage on the help component is non-existent at the moment. @jdtoombs was working on improving this before we realized that unit tests were out of scope for his role, not sure if he made any progress on this before he stopped working on that.

Reply:
Yes there is a large hole for coverage here.  I don't have time to resolve regrettably.